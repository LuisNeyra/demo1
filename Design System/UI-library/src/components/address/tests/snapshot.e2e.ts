import { snapIt } from '../../../test';

const component = 'address';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('correctly', '<blaze-address>Address line one</blaze-address>');

    snap(
      'the address heading',
      `<blaze-address>
        <blaze-address-heading>Address header</blaze-address-heading>
        Address line one<br />
        Address line two<br />
        Postcode
      </blaze-address>`
    );
  });
});
