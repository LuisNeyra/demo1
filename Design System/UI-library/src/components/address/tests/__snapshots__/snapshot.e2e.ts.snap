// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`address renders correctly 1`] = `"<blaze-address class=\\"hydrated\\"><!----><address class=\\"c-address\\">Address line one</address></blaze-address>"`;

exports[`address renders the address heading 1`] = `"<blaze-address class=\\"hydrated\\"><!----><address class=\\"c-address\\"> <blaze-address-heading class=\\"hydrated\\"><!----><span class=\\"c-address__heading\\">Address header</span></blaze-address-heading> Address line one<br> Address line two<br> Postcode </address></blaze-address>"`;
